### PR TITLE
Add note field

### PR DESCRIPTION
**Pull request by mkangia**
Got a small request to add note column to Hosted CCZs
FF: `MANAGE_CCZ_HOSTING`

### CODE REVIEW

File: corehq/toggles.py [hunk 1]
```diff
@@ -1785,7 +1785,7 @@ def _commtrackify(domain_name, toggle_is_enabled):
 
 MANAGE_CCZ_HOSTING = StaticToggle(
     'manage_ccz_hosting',
-    'Allow project to configure ccz hosting',
+    'Allow project to manage ccz hosting',
```
Comment (mkangia) [inline corehq/toggles.py]: tendency is to search for "manage" on the feature flag listing and hence this was not appearing as a result.

**Review by orangejenny — APPROVED** [review]

**Comment by orangejenny** [comment]
Test failures look related:

`FAIL: custom.icds.tests.serializers.test_hosted_ccz_serializer:TestHostedCCZSerializer.test_data`

and

```
ERROR: custom.icds.tests.models.test_hosted_ccz:TestHostedCCZ.test_delete_ccz
...
ValidationError: {'note': [u'This field cannot be blank.']}
```

File: custom/icds/forms.py [hunk 2]
```diff
@@ -107,7 +109,8 @@ def save(self):
             HostedCCZ.objects.create(
                 link_id=self.cleaned_data['link_id'], app_id=self.cleaned_data['app_id'],
                 version=self.cleaned_data['version'], profile_id=self.cleaned_data['profile_id'],
-                file_name=self.cleaned_data['file_name']
+                file_name=self.cleaned_data['file_name'],
+                note=self.cleaned_data['note']
```
Comment (proteusvacuum) [inline custom/icds/forms.py]: nit: missing trailing comma.
surprised stickler didn't catch that one.

**Review by orangejenny — APPROVED** [review]

**Comment by mkangia** [comment]
Thanks for the review @orangejenny @proteusvacuum 

did some back on forth on the blank/null and eventually just kept it `blank=True`

**Review by orangejenny — APPROVED** [review]

**Pull request merged**